src/sidebar/session.js

```diff
@@ -109,26 +67,18 @@ function session($http, $q, $resource, $rootScope, analytics, annotationUI, auth
       });
     }
     return lastLoad;
-  };
+  }
 
   /**
-   * @name session.dismissSidebarTutorial()
-   *
-   * @description Stores the preference server-side that the user dismissed
-   *              the sidebar tutorial, and then updates the session state.
+   * Store the preference server-side that the user dismissed the sidebar
+   * tutorial, and then updates the session state.
    */
   function dismissSidebarTutorial() {
     return store.profile.update({}, {preferences: {show_sidebar_tutorial: false}}).then(update);
   }
 
   /**
-   * @name session.update()
-   *
-   * @description Update the session state using the provided data.
-   *              This is a counterpart to load(). Whereas load() makes
-   *              a call to the server and then updates itself from
-   *              the response, update() can be used to update the client
-   *              when new state has been pushed to it by the server.
+   * Update the profile using the provided data.
```

Comment:
What's the reason for the docstring change here? The explanation in the `@description` seems useful to me.

Reply:
I've been changing the docstrings to be consistent with the format that we use elsewhere in the app:

```
/**
 * One line summary.
 *
 * Description.
 *
 * @param ...
 * @return ...
 */
```

However, I agree there was some useful info in the previous docstring. I've re-added that info with different wording.

Reply:
Ok, good to know that this is the style. If things like this can be done in separate commits it does make reviewing easier